datanode/src/storage_api.py

```diff
@@ -195,6 +181,48 @@ def GridCellMetaDataHandler(zoom, x, y):
   return _FormatResult(result)
 
 
+@webapp.route(
+  '/GridCellsMetaData/<zoom>',
+  methods=['GET', 'PUT', 'POST', 'DELETE'])
+def GridCellsMetaDataHandler(zoom):
+  """Handles the web service request for multi-grid operations.
+
+  Args:
+    zoom: zoom level in slippy tile format
+    x: x tile number in slippy tile format
```

Comment:
Are `x` and `y` accepted as arguments at all here?

Reply:
Nope, fixed.